Tests/LibWeb/Text/input/radio-button-focus-lost-no-change-event.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<input type="radio" id="radioButton">
+<script src="include.js"></script>
+<script>
+    radioButton.addEventListener("change", () => {
+      println("Change event was fired when it shouldn't have been.");
```

Comment:
Can we change the test so that it prints something if the test passes?

Something like:
```diff
diff --git a/Tests/LibWeb/Text/input/radio-button-focus-lost-no-change-event.html b/Tests/LibWeb/Text/input/radio-button-focus-lost-no-change-event.html
index 1195b4d0672..f595c7af56f 100644
--- a/Tests/LibWeb/Text/input/radio-button-focus-lost-no-change-event.html
+++ b/Tests/LibWeb/Text/input/radio-button-focus-lost-no-change-event.html
@@ -2,15 +2,20 @@
 <input type="radio" id="radioButton">
 <script src="include.js"></script>
 <script>
+    let changeEventFired = false;
     radioButton.addEventListener("change", () => {
-      println("Change event was fired when it shouldn't have been.");
+        changeEventFired = true;
     });
     asyncTest(async done => {
         radioButton.focus();
         await new Promise(resolve => setTimeout(resolve, 0));
         radioButton.blur();
         await new Promise(resolve => setTimeout(resolve, 0));
-
+        if (changeEventFired) {
+            println("FAIL: Change event was fired when it shouldn't have been.");
+        } else {
+            println("PASS: Change event was not fired");
+        }
         done();
     });
-</script>
\ No newline at end of file
+</script>
```

Reply:
Yes, should I change the equivalent `checkbox-focus-lost-no-change-event.html` in this same PR?
(the two tests look almost identical, I just copied and modified the checkbox one to make this one)

Reply:
> Yes, should I change the equivalent `checkbox-focus-lost-no-change-event.html` in this same PR?

Yes please, that should be fine as a separate commit in this PR.

Reply:
Done.